src/java.desktop/share/classes/javax/swing/JScrollBar.java

```diff
@@ -760,6 +760,9 @@ public void stateChanged(ChangeEvent e)   {
      * rigid along the other axis.
      */
     public Dimension getMinimumSize() {
+        if (isMinimumSizeSet()) {
+            return super.getMinimumSize();
+        }
```

Comment:
This looks like it has potential to cause many compatibility problems.
I think it better to simply update the Jscrollbar spec on these methods to reflect what JScrollbar does.

Reply:
OK. Fair enough...Updated the spec of `getMinimumSize ` and `getMaximumSize` to mention this behaviour change as JScrollBar doesn't override `setMinimumSize` and `setMaximumSize`

Reply:
CSR added https://bugs.openjdk.org/browse/JDK-8317857